plugin/pkg/auth/authenticator/request/x509/x509.go

```diff
@@ -104,7 +105,13 @@ var DNSNameUserConversion = UserConversionFunc(func(chain []*x509.Certificate) (
 
 // EmailAddressUserConversion builds user info from a certificate chain using the first EmailAddress on the certificate
 var EmailAddressUserConversion = UserConversionFunc(func(chain []*x509.Certificate) (user.Info, bool, error) {
```

Comment:
I take it you have modified that code in your own fork, and are contributing part of a fix back upstream?